### PR TITLE
feature 619 initialcommit for simulated UI Generator output for Vendi…

### LINKED ISSUE #500: [Nox.Ui] Cryptocash.Ui Phase1 integrate Nox.Types into Ui

### LINKED ISSUE #619: [Nox.Ui] Cryptocash.Ui Initial Add Search and DisplayResults so that we can agree where to generate content from config

### PR DESCRIPTION
**Pull request by DanielHodgeGH**
PR for initial simulated UI Generator output - displaying and searching VendingMachines

- Please in this instance review the Generated folder on UI
- Just focused on VendingMachine list for now 
- Current problem with Streetaddress on VendingMachine - so have commented out for now have created ticket: https://github.com/NoxOrg/Nox.Generator/issues/850
- Using Application.Server.VendingMachine entity for now as the main area to define api access and ui properties until the new meta generator entity is ready - just to get the app working

NOTE: test data can be found at https://github.com/NoxOrg/Nox.Generator/issues/688

### CODE REVIEW

File: samples/Cryptocash.Ui/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Data/ApiSetting/ApiOrder.cs
Comment (anna-naumova25) [inline samples/Cryptocash.Ui/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Data/ApiSetting/ApiOrder.cs:1]: Do we really need 'Api' prefix for ApiSetting, ApiOrder, ApiPaging etc?

File: samples/Cryptocash.Ui/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Domain.BankNote.g.cs
Comment (anna-naumova25) [inline samples/Cryptocash.Ui/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Domain.BankNote.g.cs:1]: I think we should use Application.Dto.BankNoteDto, Application.Dto.BankNoteCreateDto, Application.Dto.BankNoteUpdateDto. Those entities are designed to communicate with CRUD API. Moreover we can just make Nox render the same models in Cryptocash.Ui. So for now I think we can copy those Dto models from Cryptocash.Api.

**Review by anna-naumova25 — DISMISSED** [review]
Looks good, I've just added some small comments. @DanielHodgeGH we can always have a call if you want to discuss anything else from this PR.

**Review by rochar — DISMISSED** [review]

File: samples/Cryptocash.Ui/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Domain.BankNote.g.cs
Comment (DanielHodgeGH) [reply samples/Cryptocash.Ui/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Domain.BankNote.g.cs:1]: Have replaced all with Dto entities now - have left the entites in Cryptocash.Ui for now and not used Cryptocash.Api entites directly until we create the meta entities with additional Ui info and are ready to merge into the generator stage

File: samples/Cryptocash.Ui/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Data/ApiSetting/ApiOrder.cs
Comment (DanielHodgeGH) [reply samples/Cryptocash.Ui/Generated/Nox.Generator/Nox.Generator.NoxCodeGenerator/Data/ApiSetting/ApiOrder.cs:1]: Have replaced with new ApiSettings more inline with design yaml - userinterface settings - will eventually be replaced by new meta ui entity but until then will leave here

**Review by anna-naumova25 — APPROVED** [review]

**Review by rochar — APPROVED** [review]

**Pull request merged**